.github/workflows/test-e2e.yml

```diff
@@ -91,8 +97,7 @@ jobs:
       - name: Install k3d
         run: wget -qO - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | TAG=$K3D_VERSION bash
       - name: Provision SKR cluster
-        run: |
-          k3d cluster create skr -p 10080:80@loadbalancer -p 10443:443@loadbalancer --k3s-arg '--disable=traefik@server:0'
+        run: k3d cluster create skr -p 10080:80@loadbalancer -p 10443:443@loadbalancer --image rancher/k3s:v$K8S_VERSION-k3s2 --k3s-arg '--disable=traefik@server:0'
       - name: Provision KCP cluster
```

Comment:
Please also adjust the k8s version for the kcp cluster.

With this you could also switch to native k3d to setup the KCP cluster, since kyma provision will be dropped in future anyways

Reply:
Good idea, done. PTAL again